### PR TITLE
Review of #126

### PR DESCRIPTION
**Pull request by nud3l**
- Fixes some minor formatting issues
- Adds the Kintsugi thresholds
- Fixes the issue and redeem guides description

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -20,62 +18,46 @@
...
 
 * **Further Reading**
 
-
 * [Roadmap](about/roadmap.md)
 * [Research](about/research.md)
 * [Glossary](about/glossary.md)
-* [Formal Specification :arrow_upper_right:](https://spec.interlay.io/)
-
+* [Formal Specification 🡥](https://spec.interlay.io/)
```
Comment (alexeiZamyatin) [inline docs/_sidebar.md]:
```suggestion
* [Formal Specification  :notebook_with_decorative_cover:](https://spec.interlay.io/)
```

File: docs/_sidebar.md [hunk 2]
```diff
@@ -4,14 +4,12 @@
 
 * **Getting Started**
 
-
   * [Interlay 101](getting-started/interlay-101.md)
   * [Learn interBTC](getting-started/interbtc.md)
   * [Governance](getting-started/governance.md)
-  * [FAQ :arrow_upper_right:](https://interlay.notion.site/FAQ-Page-5e3019b1cfd94f6693dc186e9640e607)
-
+  * [FAQ 🡥](https://interlay.notion.site/FAQ-Page-5e3019b1cfd94f6693dc186e9640e607)
```
Comment (alexeiZamyatin) [inline docs/_sidebar.md]:
```suggestion
  * [FAQ :bookmark_tabs:](https://interlay.notion.site/FAQ-Page-5e3019b1cfd94f6693dc186e9640e607)
```

**Review by alexeiZamyatin — CHANGES_REQUESTED** [review]

File: docs/_sidebar.md [hunk 3]
```diff
@@ -20,62 +18,46 @@
...
-
   * [interbtc.js](developers/integration.md)
   * [References](developers/documentation.md)
-  
-
 
 * **Further Reading**
 
-
 * [Roadmap](about/roadmap.md)
```
Comment (alexeiZamyatin) [inline docs/_sidebar.md]: List formatting is off - pls fix :) (not sure what is wrong, probably spacing)

File: docs/_sidebar.md (hunk 1 above)
Comment (nud3l) [reply docs/_sidebar.md]: I used the error to indicate an external link. Not sure what the emoji should do?

File: docs/_sidebar.md (hunk 2 above)
Comment (nud3l) [reply docs/_sidebar.md]: I used the error to indicate an external link. Not sure what the emoji should do?

**Pull request merged**